### PR TITLE
The complete Korean language translation of keyboard shortcuts

### PR DESCRIPTION
**Pull request by sdivyanshu90**
Fixes: Keyboard Shortcuts

- I have noticed that the Korean translation of the keyboard shortcut is not complete. 
- It can be confusing and frustrating when trying to use certain keyboard shortcuts because some of the key labels are still in English.
image

Changes:
- I have translated the keyboard shortcuts into the Korean language.

I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] has no test errors (`npm run test`)
* [x] is from a uniquely-named feature branch and is up to date with the  `develop` branch.

### CODE REVIEW

**Comment by sdivyanshu90** [comment]
Hi @lindapaiste, could you review my pull request and provide feedback? Thanks!

**Comment by lindapaiste** [comment]
@sdivyanshu90 the PR looks good 👍🏻 

Hopefully someone who knows Korean can check it over and verify that the translations are correct.

File: translations/locales/ko/translations.json [hunk 1]
```diff
@@ -158,25 +158,26 @@
...
-      "FindText": "Find Text",
-      "FindNextMatch": "Find Next Match",
-      "FindPrevMatch": "Find Previous Match",
-      "IndentCodeLeft": "Indent Code Left",
-      "IndentCodeRight": "Indent Code Right",
-      "CommentLine": "Comment Line",
-      "FindNextTextMatch": "Find Next Text Match",
-      "FindPreviousTextMatch": "Find Previous Text Match",
-      "CodeEditing": "Code Editing"
+      "Tidy": "깔끔한",
```
Comment (raclim) [inline translations/locales/ko/translations.json]: Would change to "정돈하기"

File: translations/locales/ko/translations.json [hunk 2]
```diff
@@ -158,25 +158,26 @@
...
+      "FindText": "텍스트 찾기",
+      "FindNextMatch": "다음 경기 찾기",
```
Comment (raclim) [inline translations/locales/ko/translations.json]: Would change to "다음 일치 항목 찾기"

File: translations/locales/ko/translations.json [hunk 3]
```diff
@@ -158,25 +158,26 @@
...
+      "FindPrevMatch": "이전 경기 찾기",
```
Comment (raclim) [inline translations/locales/ko/translations.json]: Would change to "이전 일치 항목 찾기"

File: translations/locales/ko/translations.json [hunk 4]
```diff
@@ -158,25 +158,26 @@
...
+      "ReplaceTextMatch": "텍스트 일치 바꾸기",
```
Comment (raclim) [inline translations/locales/ko/translations.json]: Would change to "텍스트 일치 항목 바꾸기"

File: translations/locales/ko/translations.json [hunk 5]
```diff
@@ -158,25 +158,26 @@
...
+      "IndentCodeLeft": "코드 왼쪽 들여쓰기",
+      "IndentCodeRight": "코드 오른쪽 들여쓰기",
+      "CommentLine": "댓글 라인",
```
Comment (raclim) [inline translations/locales/ko/translations.json]: Would change to "주석 처리"

File: translations/locales/ko/translations.json [hunk 6]
```diff
@@ -158,25 +158,26 @@
...
+      "FindNextTextMatch": "다음 텍스트 일치 찾기",
```
Comment (raclim) [inline translations/locales/ko/translations.json]: Would change to "다음 텍스트 일치 항목 찾기"

File: translations/locales/ko/translations.json [hunk 7]
```diff
@@ -158,25 +158,26 @@
...
+      "FindPreviousTextMatch": "이전 텍스트 일치 찾기",
```
Comment (raclim) [inline translations/locales/ko/translations.json]: Would change to "이전 텍스트 일치 항목 찾기"

File: translations/locales/ko/translations.json [hunk 8]
```diff
@@ -158,25 +158,26 @@
   },
   "KeyboardShortcuts": {
     "Title": " 단축키",
-    "ShortcutsFollow": "Code editing keyboard shortcuts follow",
-    "SublimeText": "Sublime Text shortcuts",
+    "ShortcutsFollow": "코드 편집 키보드 단축키는 다음과 같습니다",
+    "SublimeText": "숭고한 텍스트 단축키",
```
Comment (raclim) [inline translations/locales/ko/translations.json]: Would change to "서블라임 텍스트"

File: translations/locales/ko/translations.json [hunk 9]
```diff
@@ -158,25 +158,26 @@
...
+      "CodeEditing": "코드 편집"
     },
     "General": {
-      "StartSketch": "Start Sketch",
-      "StopSketch": "Stop Sketch",
-      "TurnOnAccessibleOutput": "Turn On Accessible Output",
-      "TurnOffAccessibleOutput": "Turn Off Accessible Output"
+      "StartSketch": "스케치 시작",
+      "StopSketch": "스케치 중지",
+      "TurnOnAccessibleOutput": "접근 가능한 출력 켜기",
```
Comment (raclim) [inline translations/locales/ko/translations.json]: Would change to "접근 가능한 출력 활성화"

File: translations/locales/ko/translations.json [hunk 10]
```diff
@@ -158,25 +158,26 @@
...
+      "TurnOffAccessibleOutput": "액세스 가능한 출력 끄기"
```
Comment (raclim) [inline translations/locales/ko/translations.json]: Would change to "접근 가능한 출력 비활성화"

File: translations/locales/ko/translations.json [hunk 11]
```diff
@@ -158,25 +158,26 @@
   },
   "KeyboardShortcuts": {
     "Title": " 단축키",
-    "ShortcutsFollow": "Code editing keyboard shortcuts follow",
-    "SublimeText": "Sublime Text shortcuts",
+    "ShortcutsFollow": "코드 편집 키보드 단축키는 다음과 같습니다",
```
Comment (raclim) [inline translations/locales/ko/translations.json:161]: Would change to "코드 편집 키보드 단축키는 다음과 같습니다"

**Review by raclim — CHANGES_REQUESTED** [review]
Overall great start! The translations mostly seem fine to me, but some of the word choices might not make sense in a coding environment. I would update some of them in the lines I noted! 

My suggested changes might not be perfect though, so open to input from others as well!

**Comment by sdivyanshu90** [comment]
Thank you for your review, @raclim and @yinhwa! I've implemented all the mentioned changes.

**Review by raclim — APPROVED** [review]
Looks great, thanks so much for your work on this!

**Pull request merged**